core/functions/serialization.go

```diff
@@ -55,6 +55,8 @@ func (function Function) Serialize(ctx context.Context) ([]byte, error) {
 	// Write version 1 data
 	writer.String(function.ExtensionName)
 	writer.String(function.ExtensionSymbol)
+	writer.String(function.SQLDefinition)
```

Comment:
We have to write these under version 2, and we can add a comment stating such as well.